src/features/Apiexplorer/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { playground_requests } from '@site/src/utils/playground_requests';
+import styles from './Dropdown.module.scss';
+
+type TDropdown = {
+  selected: string;
+  setSelected: (string) => void;
+  selected_value: string;
+  handleChange: (event, string) => void;
+};
+
+type TDropdownList = {
+  selected: string;
+  setSelected: (string) => void;
+  selected_value: string;
+  handleChange: (event, string) => void;
+  searchResults: string;
+  setIsActive: (boolean) => void;
+  setSearchResults: (string) => void;
+};
+
+const DropdownList = ({
+  selected,
+  setSelected,
+  handleChange,
+  setIsActive,
+  searchResults,
+  setSearchResults,
+}: TDropdownList) => {
+  return (
+    <div>
+      <input
+        autoFocus
+        type='text'
+        data-testid='searchInput'
+        className={styles.dropdownSearch}
+        onChange={(event) => {
+          setSearchResults(event.target.value);
+        }}
+      />
+      <div className={styles.dropdownList}>
+        <div className={styles.dropdownSelect}>
+          <span>Select API Call - Version 3</span>
+        </div>
+        <div className={styles.dropdownStart}>
+          <span>ALL CALLS</span>
+        </div>
+        {playground_requests
+          .filter((option) => {
+            return option.title.toLowerCase().includes(searchResults.toLowerCase()) ? option : null;
+          })
```

Comment:
make a new variable for this, for better readablity.
```suggestion
        const api_call_list = playground_requests
          .filter((option) => {
            return option.title.toLowerCase().includes(searchResults.toLowerCase()) ? option : null;
          });
```